conf/test.config

```diff
@@ -46,7 +46,8 @@ params {
 }
 
 process {
-    withName:'.*:FREEC_SOMATIC'{
+
+    withName:'*.:FREEC_SOMATIC'{
```

Comment:
```suggestion
    withName:'FREEC_SOMATIC'{
```

Reply:
Changed it back in the hopes it would make the controlfreec tests pass. unsure what is going on there, they run locally

Reply:
I see

Reply:
`*.` won't work anyway

Reply:
it's `.*`